ci/env.sh

```diff
@@ -12,7 +12,7 @@ if [[ -n $CI ]]; then
     export CI_BUILD_ID=$TRAVIS_BUILD_ID
     export CI_COMMIT=$TRAVIS_COMMIT
     export CI_JOB_ID=$TRAVIS_JOB_ID
-    if $TRAVIS_PULL_REQUEST; then
+    if [[ $TRAVIS_PULL_REQUEST =~ ^[0-9]+$ ]]; then
```

Comment:
This is simpler: 
```suggestion
    if [[ $TRAVIS_PULL_REQUEST != false ]]; then
```
ref: https://docs.travis-ci.com/user/environment-variables/

Reply:
wfm!